contrib/relatorio.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "from pathlib import Path\n",
+    "from collections.abc import Mapping, Iterable, Generator\n",
+    "from datetime import datetime\n",
+    "import zipfile\n",
+    "from typing import Any\n",
+    "from io import BytesIO\n",
+    "import re\n",
+    "import sqlite3\n",
+    "import pandas as pd\n",
+    "# base_path = Path(__file__).resolve().parent\n",
+    "curr_folder = os.path.abspath('')\n",
+    "base_path = Path(curr_folder).resolve().parent\n",
+    "rpa_path = base_path.parent\n",
+    "sys.path.append(str(base_path))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Defina o Path para os .zip disponibilizados pela Receita Federal\n",
+    "zip_path = None\n",
+    "\n",
+    "if zip_path is None:\n",
+    "    zip_path = os.path.join(base_path, \"data\")"
```

Comment:
Qual o sentido dessa lógica?

1. Definir `zip_path` como `None`
2. Verificar se `zip_path` é `None` (ele sempre será, pois vc acabou de definir que ele é igual a `None`)
3. Se for, mudar o valor de `zip_path`

Pq não assim?

```suggestion
    "zip_path = base_path / \"data\""
```

Reply:
Sim, boa!